meson.build

```diff
@@ -81,6 +81,7 @@ dep_x11 = dependency('x11')
 dep_x11_ext = dependency('xext')
 dep_x11_xcb = dependency('x11-xcb')
 dep_xcb = dependency('xcb')
+dep_xcb_dri2 = dependency('xcb-dri2')
```

Comment:
I think this can be moved down to the dri2 block, so it doesn't get checked if it's disabled?

Reply:
And please add it to travis in this commit too, so we keep bisectability

Reply:
The build dependency `libxcb-dri2-0-dev` needs to be added, no? In contrast, we don't explicitly list `libxext-dev` there, so we don't need to remove that at least.

Reply:
On fedora it's only one libxcb packet for all libraries.
The packages are already installed as dependency, but listing them here seems a good idea.